dds/DCPS/RTPS/Sedp.cpp

```diff
@@ -3152,6 +3155,20 @@ Sedp::signal_liveliness_unsecure(DDS::LivelinessQosPolicyKind kind)
   }
 }
 
+
+bool Sedp::send_type_lookup_request(XTypes::TypeIdentifierSeq& type_ids,
+                                    const DCPS::RepoId& reader)
+{
+  const DCPS::RepoId remote_reader = make_id(reader, ENTITYID_TL_SVC_REPLY_READER);
+  DCPS::SequenceNumber sequence = 0;
+
+  return type_lookup_request_writer_->send_type_lookup_request(type_ids,
+                                                               remote_reader,
+                                                               sequence,
+                                                               type_lookup_service_->rpc_sequence_number_,
+                                                               participant_id_)  == DDS::RETCODE_OK;
```

Comment:
```suggestion
                                                               participant_id_) == DDS::RETCODE_OK;
```
Also - I try to avoid the "arguments falling over a cliff" code formatting.  We may not have a better option here.